index.html

```diff
@@ -61,27 +61,323 @@
+        <div class="hero__container">
+          <button class="hero-btn primary-btn">Shop Now</button>
+        </div>
+        <div class="hero-heading">Featured Categories</div>
+        <div class="hero__categories">
+          <div class="item item-1">
+            <p class="item-name">DOG FOOD</p>
+          </div>
+          <div class="item item-2">
+            <p class="item-name">TOYS</p>
+          </div>
+          <div class="item item-3">
+            <p class="item-name">CAT FOOD</p>
+          </div>
+          <div class="item item-4">
+            <p class="item-name">TREATS</p>
+          </div>
+        </div>
+        <div class="hero-heading">Popular Products</div>
+        <div class="products-5">
+          <section class="component">
+            <div class="card__container">
+              <div class="card-header">
+                <img src="/images/drools-image.png" alt="" class="card-image" />
+                <span class="card-badge">20% OFF</span>
+              </div>
+              <div class="card-details">
+                <h2 class="detail-heading">Drools for Dogs</h2>
+                <p class="detail-subheading">Adult 1.2 Kg</p>
+              </div>
+              <div class="card-footer">
+                <div class="price-segment">
+                  <span class="item-price">&#x20b9; 250</span>
+                  <span class="item-price dashed">&#x20b9; 350</span>
+                </div>
+                <div class="card-icons">
+                  <i class="bi bi-heart"></i>
+                  <i class="bi bi-cart"></i>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="component">
+            <div class="card__container">
+              <div class="card-header">
+                <img
+                  src="/images/pedigree-image.png"
+                  alt=""
+                  class="card-image"
+                />
+                <span class="card-badge">20% OFF</span>
+              </div>
+              <div class="card-details">
+                <h2 class="detail-heading">Pedigree for Dogs</h2>
+                <p class="detail-subheading">Adult 1.2 Kg</p>
+              </div>
+              <div class="card-footer">
+                <div class="price-segment">
+                  <span class="item-price">&#x20b9; 250</span>
+                  <span class="item-price dashed">&#x20b9; 350</span>
+                </div>
+                <div class="card-icons">
+                  <i class="bi bi-heart"></i>
+                  <i class="bi bi-cart"></i>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="component">
+            <div class="card__container">
+              <div class="card-header">
+                <img
+                  src="/images/purepet-image.png"
+                  alt=""
+                  class="card-image"
+                />
```

Comment:
inside alt add some text about the image

Reply:
sure will add some text